monai/handlers/mlflow_handler.py

```diff
@@ -70,6 +70,13 @@ class MLFlowHandler:
         state_attributes: expected attributes from `engine.state`, if provided, will extract them
             when epoch completed.
         tag_name: when iteration output is a scalar, `tag_name` is used to track, defaults to `'Loss'`.
+        experiment_name: name for an experiment, defaults to `default_experiment`.
+        run_name: name for run in an experiment, defaults to `test_run`.
+        experiment_param: a dict recording parameters which will not change through whole experiment,
+            like torch version, cuda version and so on.
+        artifacts: paths to images that need to be recorded after a whole run.
```

Comment:
The experiment management tutorial needs to cover how these artifacts are generated because the usage is quite unclear.

Reply:
Hi @mingxin-zheng ,

I think `MLFlow` only supports local image files with provided path. So maybe it's OK to save the prediction in files then visualize here? The question I have is how to distinguish the images of different experiments if we always saving the prediction to the same path to override previous results?

Thanks.